ddl/split_region.go

```diff
@@ -118,11 +123,17 @@ func splitIndexRegion(store kv.SplittableStore, tblInfo *model.TableInfo, scatte
 	return regionIDs
 }
 
-func waitScatterRegionFinish(store kv.SplittableStore, regionIDs ...uint64) {
+func waitScatterRegionFinish(ctx context.Context, store kv.SplittableStore, regionIDs ...uint64) {
 	for _, regionID := range regionIDs {
 		err := store.WaitScatterRegionFinish(regionID, 0)
```

Comment:
Could we add the argument of `ctx` to `WaitScatterRegionFinish`? Then we can remove line132-137.

Reply:
Great, done